src/main/java/io/shiftleft/fuzzyc2cpg/ast/langc/functiondef/FunctionDef.java

```diff
@@ -53,4 +54,13 @@ public void addChild(AstNode node) {
   public void accept(ASTNodeVisitor visitor) {
     visitor.visit(this);
   }
+
+  public void setIsOnlyDeclaration(boolean isOnlyDeclaration) {
```

Comment:
It might be worth considering [Lombok](https://projectlombok.org/) if we end up writing a lot of getter/setter code.

Reply:
Yes, there's a lot of old (2013)  Java code here. I would actually suggest we gradually replace it with Scala.